activities/member-relations/index.md

```diff
@@ -8,31 +8,58 @@ This outlines the stages by which a public organization becomes a founding membe
 
-This stage is over when both parties agree on the '[Founding Membership Proposal](founding-membership-proposal.md)'.
-
 This stage may include a [founding membership workshop](founding-membership-workshop.md).
 
-## 3. Founding membership contract
+This stage is over when both parties agree to the proposal.
+
+## 3. Founding membership contract (2019 only)
+
+At this stage, public organizations become founding members by signing a founding membership contract. This contract consists of the proposal agreed in the previous stage, along with any extra legal framing required. This commits both the member and the Foundation to the mutually agreed proposal.
+
+Founding members are co-owners of the foundation and help shape its future activities and regular membership structure. 
+
+This stage is over after 2019, when the founding members have shaped the Foundation's activities and regular membership structure.
+
+### Assumptions
+
+* The Founding Membership Proposal may require extra legal framing to make it a contract.
+* This will be governed by Dutch law.
+* These contracts will not automatically roll over into regular membership contracts as these are structurally different.
 
-> Founding membership contracts are only available to founding members in 2019. From 2020, all members will have an annual membership contracts.
-At this stage, the Foundation and the public organization sign a legally binding contract, including a financial settlement and the founding membership proposal (attached as an annex).
+## 4. Membership of the Foundation (2020 onwards)
 
-This stage is over when both parties sign a founding membership contract.
+From 2020, each member has a regular membership contract.
 
+With regular membership, members become co-owners of the Foundation. This means members can vote in the general assembly and are responsible for and steer the Foundation’s operations, including priorities and annual budget for the organization. [Read more about Foundation governance].
 
-## 4. Membership of the Foundation
+All members pay membership yearly dues. [See financial model]. Failure to pay can result in termination of membership as stipulated in the [foundation bylaws]. 
 
-From 2020, each member will have an annual membership contract.
+A mediation structure will be set up to handle any disputes that may arise between members and the Foundation or between individual members.
 
-With the annual membership contract, members become co-owners of the Foundation. This includes steering and setting priorities for the Foundation and voting in the General Assembly. 
+### Assumptions
+* This contract will be governed by Dutch law.
+* Members will not pay dues through procurement or tendering procedures.
+* The procedures members use to pay their dues will differ based on local legal context -  for example, some could use a grant.
+* Members will not incur obligations to the Foundation beyond paying dues and any other obligations stipulated in the contract.
```

Comment:
```suggestion
* Members will not incur obligations to the Foundation beyond paying dues.
```
This is much more helpful if we can provide a definitive list. "Members do not incur any obligations beyond the obligations in the contract' seems more weasely (though that is more futureproofed).

Reply:
True - I guess we can always update this once we get the contract defined